content/blog/dplyr-backends/index.Rmd

```diff
@@ -0,0 +1,182 @@
+
+    ```{r}
+    dt %>% 
+      filter(if_any(cyl:wt, is.na)) %>% 
+      show_query()
+    ```
+
+-   `relocate()`, which is translated to the `j` argument of `[.data.table`:
+
+    ```{r}
+    dt %>% 
+      relocate(carb, .before = mpg) %>% 
+      show_query()
+    ```
+
+-   `rename_with()`, which is translated to `setnames()`:
+
+    ```{r}
+    dt %>% 
+      rename_with(toupper) %>% 
+      show_query()
+    ```
+
+-   `slice_min()`, `slice_max()`, `slice_head()`, `slice_tail()`, and `slice_sample()` which are translated to various `i` and `j` expressions:
+
+    ```{r}
+    dt %>% group_by(cyl) %>% slice_sample(n = 5) %>% show_query()
+    dt %>% group_by(cyl) %>% slice_head(n = 1) %>% show_query()
+    dt %>% group_by(cyl) %>% slice_min(mpg, n = 1) %>% show_query()
+    ```
+
+Thanks to [Mark Fairbanks](https://github.com/markfairbanks), dtplyr has also gained it's first translation of a tidyr function: `pivot_wider()`, which is translated to `dcast()`.
+You can expect more tidyr translations in the next release 😄.
+
+I also took this as an opportunity to thoroughly refresh the documentation so that all translated verbs now have [their own help pages](https://dtplyr.tidyverse.org/reference/index.html) that briefly describe how the translation works.
+You can read about the other minor improvements and bug fixes in the [release notes](https://github.com/tidyverse/dtplyr/releases/tag/v1.1.0).
+
+## dbplyr 2.1.0
+
+[dbplyr](https://dbplyr.tidyverse.org) translates dplyr pipelines to their SQL equivalents.
+If you're new to using dplyr and SQL together, I highly recommend Ireve Steve's rstudio::global() talk ["The dynamic duo: SQL and R](https://rstudio.com/resources/rstudioglobal-2021/the-dynamic-duo-sql-and-r/). It discusses why you might want to use dbplyr to generate SQL **and** why you should still learn SQL.
+
+The biggest change to this release is the addition of many translations for tidyr verbs like [`pivot_longer()`](https://dbplyr.tidyverse.org/reference/pivot_longer.tbl_lazy.html), [`pivot_wider()`](https://dbplyr.tidyverse.org/reference/pivot_wider.tbl_lazy.html), [`complete()`](https://dbplyr.tidyverse.org/reference/complete.tbl_lazy.html), and [`replace_na()`](https://dbplyr.tidyverse.org/reference/replace_na.tbl_lazy.html).
+These were contributed by [Maximilian Girlich](https://github.com/mgirlich), and in recognition of his sustained and substantial contributions to the package, he has been added as a package author.
+
+This release also includes major improvements to the `across()` translation, including translation of formulas (like dtplyr, `across()` can't currently use `where()`, because I don't know of a way to figure out the column types without executing the query).
+The release also includes a bunch of other minor translation improvements and bug fixes, which you can read about in the [release notes](https://github.com/tidyverse/dbplyr/releases/tag/v2.1.0).
```

Comment:
The past two sentences both have “release also includes”…

Maybe: There are also… before the “a bunch of other minor translation improvements”?